ltm.go

```diff
@@ -612,6 +616,16 @@ func (p *Monitor) UnmarshalJSON(b []byte) error {
 	return marshal(p, &dto)
 }
 
+func (p *Monitor) cleanMonitor() {
```

Comment:
I would argue against doing this here. Cleaning these strings isn't so much a function of the monitor itself but more a serialization concern before the call out to the F5. Can we move this into the marshall method after the Monitor is marshalled into the dto then munge the fields on the dto. The reasoning being that calling `cleanMonitor` from `AddMonitor` would result in altering the `send` field on the object passed in. This is probably not a side effect the caller would be expecting.

```
func (p *Monitor) MarshalJSON() ([]byte, error) {
	var dto monitorDTO
	marshal(&dto, p)
        if strings.Contains(dto.SendString, "\r\n") {
 		dto.SendString = strings.Replace(dto.SendString, "\r\n", "\\r\\n", -1)
 	}
	return json.Marshal(dto)
}